ingenialink/ethercat/dictionary.py

```diff
@@ -51,6 +56,27 @@ def _monitoring_disturbance_registers(self) -> list[EthercatRegister]:
             ),
         ]
 
+    @cached_property
+    def _safety_modules(self) -> list[DictionarySafetyModule]:
```

Comment:
I'm not sure how we can handle this. This will ony support phase I. For phase II it will be broken again.
Maybe we need to let go of the idea of using safety in xdf v2. Not sure